src/tests/FlapperUniV2.t.sol

```diff
@@ -83,22 +88,26 @@ contract FlapperUniV2Test is DssTest {
     using stdStorage for StdStorage;
 
     FlapperUniV2   public flapper;
+    FlapperUniV2   public linkFlapper;
     MockMedianizer public medianizer;
 
     address     constant  LOG           = 0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F;
     address     immutable DAI_JOIN      = ChainlogLike(LOG).getAddress("MCD_JOIN_DAI");
     address     immutable SPOT          = ChainlogLike(LOG).getAddress("MCD_SPOT");
     address     immutable DAI           = ChainlogLike(LOG).getAddress("MCD_DAI");
     address     immutable MKR           = ChainlogLike(LOG).getAddress("MCD_GOV");
+    address     immutable USDC          = ChainlogLike(LOG).getAddress("USDC");
+    address     immutable LINK          = ChainlogLike(LOG).getAddress("LINK");
     address     immutable PAUSE_PROXY   = ChainlogLike(LOG).getAddress("MCD_PAUSE_PROXY");
     VatLike     immutable vat           = VatLike(ChainlogLike(LOG).getAddress("MCD_VAT"));
     VowLike     immutable vow           = VowLike(ChainlogLike(LOG).getAddress("MCD_VOW"));
     EndLike     immutable end           = EndLike(ChainlogLike(LOG).getAddress("MCD_END"));
     SpotterLike immutable spotter       = SpotterLike(ChainlogLike(LOG).getAddress("MCD_SPOT"));
+    PipLike     immutable linkPip       = PipLike(ChainlogLike(LOG).getAddress("PIP_LINK"));
```

Comment:
We might want to use the medianizer instead of the osm.

Reply:
Or to keep the tests symmetrical and easier to read I think it would be fine to use another mock medianizer for LINK.

Reply:
Alright, changed to using mock medainizer also for LINK.
It is also getting offboarded now so relying on the oracle is not a good idea.